<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>384</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>384</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>384</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MediaPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/ico/ico/ico.ico</normaloff>:/ico/ico/ico.ico</iconset>
  </property>
  <widget class="QLabel" name="labelFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Song file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderVolum">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderProgres">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolum">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgres">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMute">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClr">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CLR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShuffle">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoop">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
